edx-platform/nau-basic/lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -166,6 +166,9 @@
                             {% endfor %}
                         </td>
                     </tr>
+                    <tr>
```

Comment:
Is this block necessary?

Reply:
I added that block to be able to modify the email footer with other templates. For example: https://github.com/fccn/nau-themes/blob/da13ac2e1f3eec5dd0d622c01f529fb3c3669c44/edx-platform/nau-basic/lms/templates/schedules/edx_ace/recurringnudge_day3/email/body.html#L56

If I don't add it, I can only modify the content, which is inside the email box.

Reply:
If I don't add it, I can only modify the content, which is inside the email box and before the call to action (the button).

![image](https://github.com/user-attachments/assets/60ef5482-a8c8-4ed1-ba49-7fe976ceaca9)

Reply:
Okey, thanks for the explanation!